pkg/bucketaccessrequest/bucketaccessrequest.go

```diff
@@ -119,7 +131,7 @@ func (b *bucketAccessRequestListener) provisionBucketAccess(ctx context.Context,
 		UID:       sa.ObjectMeta.UID}
 	// bucketaccess.Spec.MintedSecretName - set by the driver
 	bucketaccess.Spec.PolicyActionsConfigMapData, err = util.ReadConfigData(b.kubeClient, bucketAccessClass.PolicyActionsConfigMap)
-	if err != nil {
+	if err != nil && err != util.ErrNilConfigMap {
 		return err
 	}
 	//  bucketaccess.Spec.Principal - set by the driver
```

Comment:
For this and the other set by driver comment -- it's really set by the sidecar, right?

Reply:
correct.